app/message-attachments/client/stylesheets/messageAttachments.css

```diff
@@ -83,6 +83,8 @@ html.rtl .attachment {
 		margin-top: 4px;
 
 		line-height: 0;
+
+		image-orientation: from-image;
```

Comment:
@RocketChat/frontend is this something we can rely on?

Reply:
no